packages/web-components/package.json

```diff
@@ -47,42 +46,28 @@
     "test-firefox:verbose": "karma start karma.conf.js --browsers=FirefoxHeadless --single-run --coverage --reporter=mocha"
   },
   "devDependencies": {
-    "@storybook/html": "6.5.5",
-    "@types/chai": "^4.2.11",
-    "@types/chai-spies": "^1.0.1",
-    "@types/karma": "^5.0.0",
-    "@types/mocha": "^7.0.2",
-    "chai": "^4.2.0",
+    "@types/chai": "4.3.3",
+    "@types/chai-spies": "1.0.3",
+    "@types/karma": "6.3.3",
+    "@types/mocha": "7.0.2",
+    "chai": "4.3.6",
     "chai-spies": "1.0.0",
-    "circular-dependency-plugin": "^5.0.2",
-    "eslint-plugin-import": "^2.22.1",
-    "esm": "^3.2.25",
-    "ignore-loader": "^0.1.2",
-    "istanbul": "^0.4.5",
-    "istanbul-instrumenter-loader": "^3.0.1",
+    "esm": "3.2.25",
+    "ignore-loader": "0.1.2",
+    "istanbul": "0.4.5",
+    "istanbul-instrumenter-loader": "3.0.1",
     "jsdom-global": "3.0.2",
-    "karma": "^6.3.0",
-    "karma-chrome-launcher": "^3.1.0",
-    "karma-coverage": "^2.0.2",
-    "karma-coverage-istanbul-reporter": "^3.0.0",
+    "karma": "6.4.0",
+    "karma-chrome-launcher": "3.1.1",
+    "karma-coverage": "2.2.0",
+    "karma-coverage-istanbul-reporter": "3.0.3",
     "karma-firefox-launcher": "^1.3.0",
-    "karma-mocha": "^2.0.1",
-    "karma-mocha-reporter": "^2.2.5",
-    "karma-source-map-support": "^1.4.0",
-    "karma-sourcemap-loader": "^0.3.7",
-    "karma-webpack": "^4.0.2",
-    "mocha": "^7.1.2",
-    "rollup": "^2.41.0",
-    "rollup-plugin-commonjs": "^10.1.0",
```

Comment:
all rollup was propagated to single version policy approach, karma will be moved in another PR